integration/integration_test.go

```diff
@@ -106,6 +106,12 @@ func TestRun(t *testing.T) {
 		"Dockerfile_test_multistage": {"file=/foo2"},
 	}
 
+	// Map for additional flags
```

Comment:
In the test somewhere it would be good to make sure there is only one additional layer. Any ideas?

Reply:
I added a test to make sure docker built and kaniko built images have the same number of layers (except for specified offsets for the --single-snapshot ones)

But it looks like it found errors in some of the other tests so I'll try and find those bugs as well